packages/block-editor/src/components/spacing-panel-control/index.js

```diff
@@ -1,23 +1,27 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
 import { PanelBody } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import InspectorControls from '../inspector-controls';
-import useEditorFeature from '../use-editor-feature';
 
 export default function SpacingPanelControl( { children, ...props } ) {
-	const isSpacingEnabled = useEditorFeature(
-		'__experimentalEnableCustomSpacing'
-	);
-	// TODO: Renable the proper guard
-	// This is temporarily disabled for testing purposes.
-	if ( isSpacingEnabled ) return null;
-	// if ( ! isSpacingEnabled ) return null;
+	const isSpacingEnabled = useSelect( ( select ) => {
+		const { getSettings } = select( 'core/block-editor' );
+		return get( getSettings(), '__experimentalEnableCustomSpacing' );
+	} );
+
+	if ( ! isSpacingEnabled ) return null;
```

Comment:
Should this rely on "useFeature"?  cc @gziolo

Reply:
We can decide later, my take on it is what fits Global Styles, it should be considered editor features so site owners could decide what to allow for user with different roles.